packages/paste-core/components/tabs/src/TabPanels.tsx

```diff
@@ -1,17 +1,31 @@
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+import {TabsContext} from './TabsContext';
 
 export interface TabPanelsProps {
   children: React.ReactNode;
+  element?: BoxProps['element'];
 }
 
-const TabPanels = React.forwardRef<HTMLDivElement, TabPanelsProps>(({children, ...props}, ref) => {
-  return (
-    <Box {...safelySpreadBoxProps(props)} width="100%" ref={ref}>
-      {children}
-    </Box>
-  );
-});
+const TabPanels = React.forwardRef<HTMLDivElement, TabPanelsProps>(
+  ({children, element = 'TAB_PANELS', ...props}, ref) => {
+    const {orientation} = React.useContext(TabsContext);
+    return (
+      <Box {...safelySpreadBoxProps(props)} element={`${orientation.toUpperCase()}_${element}`} width="100%" ref={ref}>
```

Comment:
Maybe a shared package util function to help with these element names:

```js
function getElementName(orientation: 'horizontal' | 'vertical', element?: string, fallback: string) {
  if (element == null) {
   return `${orientation.toUpperCase()}_${fallback}`;
  }
  return element;
}
```

And this code would be 
```js
element={getElementName(orientation, element, 'TAB_PANELS')}
```

Reply:
Yeah that sounds like a good idea. Is that something that we should provide as a named export, or have it be "internal" to the `/tabs`?

Reply:
Internal to tabs is fine